.github/workflows/publish.yaml

```diff
@@ -54,11 +54,37 @@ jobs:
       - name: Save package for following jobs
         uses: actions/upload-artifact@v3
         with:
-          name: dist-macos-${{ matrix.python-version }}
+          name: dist-macos-intel-${{ matrix.python-version }}
+          path: dist
+
+  build-macos-arm:
+    runs-on: macos-latest-xlarge
```

Comment:
is there a reason for using xlarge for arm and large for intel?

Reply:
As I understand Githubs documentation: large --> Intel. xlarge --> Arm https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#:~:text=The%20following%20machines%20sizes%20are%20available%20for%20macOS%20larger%20runners.

Reply:
Hm, agree, in the link I sent below I understood otherwise